docs.go

```diff
@@ -19,10 +19,10 @@ func splitDocs(docs string) (title, desc string) {
 	return
 }
 
-// RapiDocHandler renders documentation using RapiDoc.
-func RapiDocHandler(pageTitle string) Handler {
-	return func(c *gin.Context) {
-		c.Data(200, "text/html", []byte(fmt.Sprintf(`<!doctype html>
+// RapiDocTemplate is the template used to generate the RapiDoc.  It needs two args to render:
+// 1. the title
+// 2. the path to the openapi.yaml file
+var RapiDocTemplate = `<!doctype html>
```

Comment:
Should these be mutable? If not, we may want a func to return the string. Could also just be a func that renders the template, so you can take in args to describe the inputs instead of using comments.

Reply:
Yeah, It'd be better to just export a function that would take args, instead of forcing a consumer of the module to know about the internals of a template.

I initially took the "export templates" approach because it helped me minimize the changes in all of the docs-Handlers in `docs.go`

Reply:
functionized this: 2852753